ts/state/ducks/userConfig.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * This slice is intended for the user configurable settings for the client such as appearance, autoplaying of links etc.
+ * Anything setting under the cog wheel tab.
+ */
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+export interface UserConfigState {
+  audioAutoplay: boolean;
+}
+
+export const initialUserConfigState = {
+  audioAutoplay: false,
+};
+
+const userConfigSlice = createSlice({
+  name: 'userConfig',
+  initialState: initialUserConfigState,
+  reducers: {
+    updateUserConfig(state: UserConfigState, action: PayloadAction<UserConfigState>) {
+      return {
+        ...state,
+        audioAutoplay: true,
```

Comment:
I guess you will need to remove that line once you fetch the user config from db